patches/minecraft/net/minecraft/fluid/Fluid.java.patch

```diff
@@ -18,5 +26,36 @@
 +   @Override
 +   public java.util.Set<net.minecraft.util.ResourceLocation> getTags() {
 +      return reverseTags.getTagNames();
++   }
++
++   /**
++    * Creates the fluid attributes object, which will contain all the extended values for the fluid that aren't part of the vanilla system.
++    * Do not call this from outside. To retrieve the values use {@link Fluid#getAttributes()}
++    */
++   protected net.minecraftforge.fluids.FluidAttributes createAttributes(Fluid fluid)
++   {
++      if (fluid instanceof EmptyFluid)
++         return net.minecraftforge.fluids.FluidAttributes.builder("water", null, null)
```

Comment:
Shouldn't this ID be `"empty"`, not `"water"`?
```suggestion
+         return net.minecraftforge.fluids.FluidAttributes.builder("empty", null, null)
```

Reply:
Yes. Nice catch.